### PR TITLE
line.js update (line charts generated from templates)

### PR DESCRIPTION
**Pull request by mistergone**
This PR replaces the existing `line.js` with a new file that utilizes the new code in `cfpb-chart-builder`.

## Changes
- `cfpb-chart-builder` code in `line.js`
- new info in `charts.js`

## Review
- @cfarm @marteki 

[Preview this PR without the whitespace changes](?w=0)

### CODE REVIEW

**Comment by cfarm** [comment]
when i build this locally the colors are wrong:

screen shot 2016-12-12 at 4 38 28 pm

looks like the adjusted data line is showing up gray (i made that the default which will appear if a color/class is not getting properly applied)

**Comment by cfarm** [comment]
it also looks like the wrong path/line element gets the unadjusted class in each line graph:

screen shot 2016-12-12 at 4 41 15 pm

adjusted should be dark green above and unadjusted lighter green

File: src/static/js/line.js [hunk 1]
```diff
@@ -1,248 +1,195 @@
...
-          d.seasonal = true;
-        } else {
-          d.seasonal = false;
+    var props = {
+      data: data,
+      selector: '#' + chartInfo.elementID,
+      yAxisTickFactor: chartInfo.yAxisTickFactor,
+        lineSets: {
+          'Unadjusted': {
+            classes: 'line line__adjusted',
```
Comment (cfarm) [inline src/static/js/line.js]: should be `line_unadjusted`, also typo below

File: src/static/js/line.js [hunk 2]
```diff
@@ -1,248 +1,195 @@
...
+            showInLegend: true
+          },
+          'Unadjusted Projected': {
+            classes: 'line line__adjusted line__projected',
```
Comment (cfarm) [inline src/static/js/line.js]: classes should be `'line line__unadjusted line__projected'`

File: src/static/js/line.js [hunk 3]
```diff
@@ -1,248 +1,195 @@
...
+            showInLegend: false
+          },
+          'Seasonally Adjusted': {
+            classes: 'line line_unadjusted',
```
Comment (cfarm) [inline src/static/js/line.js]: should be `'line line_adjusted'`

**Comment by mistergone** [comment]
@cfarm - A missing underscore and two properties swapped. Fixing those now!

File: src/static/js/line.js [hunk 4]
```diff
@@ -1,248 +1,195 @@
...
+            showInLegend: true
+          },
+          'Seasonally Adjusted Projected': {
+            classes: 'line line_unadjusted line__projected',
```
Comment (cfarm) [inline src/static/js/line.js]: should be `'line line_adjusted line__projected'`

**Review by cfarm — CHANGES_REQUESTED** [review]

**Comment by cfarm** [comment]
These are looking great! I am testing the release task now.

**Comment by cfarm** [comment]
Release task is working. all the non-line graphs save for 2 auto bar charts are missing from charts.js though, so not all the graphs get compiled. we'll need to clean up that file with all the jawns

**Comment by mistergone** [comment]
This PR is going to be closed and superseded by a better PR!

**Pull request closed without merging**